src/sharedInterfaces/webviewRoutes.ts

```diff
@@ -6,4 +6,5 @@
 export enum WebviewRoute {
 	tableDesigner = 'tableDesigner',
 	connectionDialog = 'connectionDialog',
+	executionPlan = 'executionPlan'
```

Comment:
maybe 'executePlanDocument' is better.  I'm worried that "executionPlan" is too generic and will be confused with the execution plan graph that is embedded in the query editor.  Please throughout the code try to make it clear which part of the code is generic to all the execution plan code paths, and the parts specific to SQLPLAN documents.

Reply:
changed it